src/app/public/modules/input-box/input-box.component.spec.ts

```diff
@@ -144,26 +162,24 @@ describe('Input box component', () => {
     });
   });
 
-  it('should render the label and input elements in the expected locations', () => {
+  it('should handle SkyThemeService not provided', () => {
+    TestBed.overrideProvider(SkyThemeService, {
```

Comment:
This is going to ensure that the component handles an undefined `SkyThemeService`, but it doesn't test the case where Angular has no provider for `SkyThemeService`. I think I'd maybe flip these two tests where the `TestBed` doesn't have `SkyThemeService` provided at all, but the test where we ensure providing it has the desired effect uses `overrideComponent` to inject `SkyThemeService`.

Reply:
I had to wrap the default and modern tests in their own `describe` blocks and create separate `TestBed`s to get it to work. I also put the duplicate testing module configuration in an `InputBoxFixturesModule`.